LivingCity/command_line_options.ini

```diff
@@ -4,11 +4,16 @@ USE_CPU=false
 NETWORK_PATH=berkeley_2018/new_full_network/
 USE_JOHNSON_ROUTING=false
 USE_SP_ROUTING=true
-USE_PREV_PATHS=true
+USE_PREV_PATHS=false
 LIMIT_NUM_PEOPLE=256000
 ADD_RANDOM_PEOPLE=false
 NUM_PASSES=1
 TIME_STEP=0.5
 START_HR=5
 END_HR=12
-SHOW_BENCHMARKS=true
+SHOW_BENCHMARKS=false
+A=0.557040909258405
+B=2.9020578588167
+T=0.5433027817144876
+s_0=1.3807498735425845
+ 
```

Comment:
I think you should just add the OD filename enhancement into this one as well. We might as well get that into the functionality as soon as possible.